cfg/rke-cis-1.24/master.yaml

```diff
@@ -298,6 +300,7 @@ groups:
               compare:
                 op: bitmask
                 value: "600"
+        type: manual
```

Comment:
also here.

What's the reasoning behind setting the type as "manual" for an automated check? Ideally, we should avoid doing this.

Reply:
Same response as above